pages/implementation/start-using-avo-functions.mdx

```diff
@@ -41,45 +41,22 @@ If a source doesn't already exist for your app or website, click "Add Source" to
 
 ### Configure destinations
 
-In Avo there are two high level options when it comes to destinations. We'll cover the difference and how you can set your destination up in your Avo workspace:
-1. [Option 1: Self-managed analytics SDK instances (custom destination)](#option-1-initialize-self-managed-analytics-sdk-instance)
-2. [Option 2: Fully Avo managed analytics SDK](#option-2-fully-avo-managed-destinations)
-3. [Configure in your Avo workspace](#configure-in-your-avo-workspace)
+1. [Provide the destination interface for each analytics destination](#implement-the-destination-interface)
+2. [Configure in your Avo workspace](#configure-in-your-avo-workspace)
 
-#### Option 1: Initialize self-managed analytics SDK instance (custom destination)
+#### Implement the destination interface
 
-Initialize the destination (e.g. SDK) on your own and pass it to the Avo SDK. This is what we call a [Custom Destination](../implementation/start-custom-destination).
-
-> 💡  **Self-managed analytics SDK instances are great for:**
->
-> A/ If you have _**existing tracking**_ and you intend to use Avo Functions alongside an analytics SDK that's already initialized in your app and receiving data.
->
-> B/ If you're sending events to your own endpoint.
->
-> C/ If you want to try the latest analytics tool that Avo doesn't yet officially support. If that happens, [let us know](../help/troubleshooting)!
+Initialize the destination (e.g. SDK) on your own and pass it to the Avo SDK, by implementing the [destination interface](../implementation/destinations) and providing it to the Avo generated code.
 
 _Step 1: **Implement a bridge interface** between your existing analytics SDK instance and the Avo Functions._
 
 - This enables you to use the same analytics SDK instance across all tracking calls (using the same session IDs and any other details that your analytics instance keeps track of), so that your users will be identified in the same way across both the tracking implemented with Avo Functions and your existing analytics implementation.
 - This is a one-time setup per codebase.
-- [See example of a Python -> Segment bridge interface.](../implementation/start-custom-destination#python)
+- [See example of a bridge interface in JavaScript.](../implementation/reference/javascript#destination-interface-example)
```

Comment:
```suggestion
- [See example of a callback interface in JavaScript.](../implementation/reference/javascript#destination-interface-example)
```

Reply:
destination interface 🤔

Reply:
Right! Suggestion on how we talk about this:
- High level: It's a _destination interface_
- More detail: It's a _destination interface_ which provides a set of _callbacks_

Whatever we end up going with, we should stick to it and make sure it's used correctly throughout the app and the docs, to avoid making a complex thing even more confusing to people 😅